spatialos-sdk/src/worker/component.rs

```diff
@@ -53,6 +53,60 @@ where
     fn get_response_command_index(response: &Self::CommandResponse) -> u32;
 }
 
+/// Additional parameters for sending component updates.
+///
+/// Additional parameters passed to [`Connection::send_component_update`]. Note that
+/// all parameters are kept private and the struct can only be initialized with
+/// default values in order to make it possible to add new parameters without a
+/// breaking change.
+///
+/// If you would like to use a method-chaining style when initializing the parameters,
+/// the [tap] crate is recommended. The examples below demonstrate this.
+///
+/// # Parameters
+///
+/// * `loopback: false` allow the update to be sent back to the worker without waiting
```

Comment:
Given the implementation: `loopback: true` allows for the short circuiting behaviour. 😉

Reply:
Ah sorry, the wording there is confusing. I only meant to indicate that the default value is `false`. I'll try to tweak the wording to make it more clear!